### PR TITLE
Tidying / formatting of poms in nucleus module

### PR DESCRIPTION
**Pull request by arjantijms**
Signed-off-by: arjantijms <arjan.tijms@gmail.com>

### CODE REVIEW

File: nucleus/core/bootstrap/pom.xml [hunk 1]
```diff
@@ -68,14 +68,9 @@
             <artifactId>osgi.cmpn</artifactId>
             <scope>provided</scope>
         </dependency>
-        <dependency>
-            <!--
```
Comment (dmatej) [inline nucleus/core/bootstrap/pom.xml:72]: I would start the comment on the new line ...

File: nucleus/core/bootstrap/pom.xml [hunk 2]
```diff
@@ -113,12 +108,8 @@
                     </archive>
                 </configuration>
             </plugin>
-            <plugin>
-                <!-- 
-                    We package simple-glassfish-api classes in this bundle 
-                    for embedded scenario, we expect user to just bootstrap the runtime by installing 
-                    glassfish.jar alone 
-                -->
+            <plugin><!-- We package simple-glassfish-api classes in this bundle for embedded scenario, we expect user to
```
Comment (dmatej) [inline nucleus/core/bootstrap/pom.xml:111]: Here again, NL. Also groupId is not required if it has this value.

File: nucleus/admin/util/pom.xml [hunk 3]
```diff
@@ -79,15 +79,9 @@
...
-        <dependency>
-            <!-- 
-                We compile against the security module but we don't want  to create a hard 
-                dependency because that would drag it into the nucleus distribution 
-                which we'd like to avoid. 
-                
-                The actual uses of SSLUtils from the security module 
-                are instantiated just-in-time only when needed, rather than being injected. 
-            -->
+        <dependency><!-- We compile against the security module but we don't want to create a hard dependency because that
```
Comment (dmatej) [inline nucleus/admin/util/pom.xml:82]: New line

File: nucleus/featuresets/atomic/pom.xml [hunk 4]
```diff
@@ -57,41 +57,41 @@
...
+            <exclusions>
                 <exclusion>
                     <groupId>*</groupId>
                     <artifactId>*</artifactId>
                 </exclusion>
             </exclusions>
         </dependency>
-        <!-- We install fileinstall bundle which monitors modules
-              dir and autodeploy dir for addition/removal/updation of bundles -->
+        <!-- We install fileinstall bundle which monitors modules dir and autodeploy dir for addition/removal/updation of
```
Comment (dmatej) [inline nucleus/featuresets/atomic/pom.xml:89]: Weird formatting here ...

File: nucleus/featuresets/atomic/pom.xml [hunk 5]
```diff
@@ -962,16 +962,8 @@
...
-            <groupId>org.glassfish.jersey.media</groupId>
-            <artifactId>jersey-media-json-processing</artifactId>
-            <exclusions>
-                <exclusion>
-                    <groupId>*</groupId>
-                    <artifactId>*</artifactId>
-                </exclusion>
-            </exclusions>
-        </dependency>-->
+        <!-- <dependency> <groupId>org.glassfish.jersey.media</groupId> <artifactId>jersey-media-json-processing</artifactId>
```
Comment (dmatej) [inline nucleus/featuresets/atomic/pom.xml:965]: Fix formatting or delete it.

File: nucleus/featuresets/nucleus/pom.xml [hunk 6]
```diff
@@ -216,24 +216,10 @@
...
-        </dependency>
-        <dependency>
-            <groupId>com.sun.xml.bind</groupId>
-            <artifactId>jaxb-osgi</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>org.glassfish.metro</groupId>
-            <artifactId>webservices-extra-jdk-packages</artifactId>
-        </dependency>-->
+        <!-- <dependency> <groupId>jakarta.xml.bind</groupId> <artifactId>jakarta.xml.bind-api</artifactId> <exclusions>
```
Comment (dmatej) [inline nucleus/featuresets/nucleus/pom.xml:219]: Fix formatting or delete it.

**Review by dmatej — CHANGES_REQUESTED** [review]
If you would fix those comments I pointed at, I would approve it ;-)

**Review by dmatej — APPROVED** [review]

**Review by jGauravGupta — APPROVED** [review]

**Pull request merged**